docs/docs/getting-started/03-installation-and-launch.md

```diff
@@ -15,6 +15,10 @@ For Ubuntu users, make sure to follow
 the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to create the `docker` group
 and add your user.
 
+Also, note that Docker desktop for Linux is currently **not** supported as it uses a custom context which can not be 
```

Comment:
nitpicks:
1. Feel like this could almost be it's separate warning
2. the "custom context" part is oddly specific, mabye we skip that and just state its not supported because it does not allow ....

Also after your investigations, even if someone has docker desktop, does just installing docker engine fix it or not?

Reply:
Then maybe that is useful information on a high level too, i.e. if someone has desktop installed, it's possible to just install engine but we recommend removing desktop since to ensure full compatibility with launcher

Reply:
I rephrased a bit and added this to the information. Might be a little too long at the moment